types/slate/index.d.ts

```diff
@@ -1131,9 +1131,12 @@ export interface Plugin {
     schema?: SchemaProperties;
 }
 
+export type Plugins = Plugin | Plugin[] | PluginStack;
+export interface PluginStack extends Array<Plugins> {}
+
 export interface EditorProperties {
     onChange?: (change: { operations: Immutable.List<Operation>, value: Value }) => void;
-    plugins?: Plugin[];
+    plugins?: PluginStack;
```

Comment:
PluginStack is good but Plugins seems to be more intuitive here?

Reply:
Sure, why not.

Reply:
Ugh, I got this wrong, sorry, I'm switching contexts a lot right now ;). Will fix asap.